workflows/general/samtools-merge.wdl

```diff
@@ -12,20 +12,18 @@ workflow samtools_merge {
         outputs: {
             merged_bam: "The BAM resulting from merging all the input BAMs"
         }
+        allowNestedInputs: true
     }
     parameter_meta{
         bams: "BAMs to merge into a final BAM"
         max_length: "Maximum number of BAMs to merge before using iteration"
-        max_retries: "Number of times to retry in case of failure"
         prefix: "Prefix for output BAM."
         use_all_cores: "Use all cores? Recommended for cloud environments. Not recommended for cluster environments."
-
     }
     input {
         Array[File] bams
         Int max_length = 100
-        Int? max_retries
-        String prefix = basename(bams[0], ".bam")
+        String prefix = basename(bams[0], ".bam")  # TODO is this a sane default? Or should be required?
```

Comment:
That's a good question. It likely should just be a required parameter. But it's fine to leave the `TODO` for now.